src/lib/components/SgtSvgIcon/SgtSvgIcon.vue

```diff
@@ -15,32 +15,34 @@
 * please email opensource@seagate.com.
 -->
 <template>
-  <v-tooltip left :disabled="!tooltip">
-    <template v-slot:activator="{ on, attrs }">
-      <div
-        class="icon-container"
-        v-if="icon"
-        :class="[hoverIcon ? 'action-btn' : '', { disabled }]"
-        v-bind="attrs"
-        v-on="on"
-      >
-        <img
-          :src="require(`@/assets/icons/${icon}`)"
-          @click="$emit('click')"
-          class="action-btn-block"
-          alt="logo"
-        />
-        <img
-          v-if="hoverIcon"
-          :src="require(`@/assets/icons/${hoverIcon}`)"
-          @click="$emit('click')"
-          class="action-btn-hover"
-          alt="logo"
-        />
-      </div>
-    </template>
-    <span>{{ tooltip }}</span>
-  </v-tooltip>
+  <div class="sgt-icon-container">
+    <v-tooltip left :disabled="!tooltip">
+      <template v-slot:activator="{ on, attrs }">
+        <div
+          class="icon-container"
```

Comment:
Since you have given a wrapping div this class name "icon-container" and its styles can be removed.

Reply:
Done